github_actions/lib/dependabot/github_actions/file_parser.rb

```diff
@@ -20,13 +20,6 @@ class FileParser < Dependabot::FileParsers::Base
 
       require "dependabot/file_parsers/base/dependency_set"
 
-      GITHUB_REPO_REFERENCE = %r{
-        ^(?<owner>[\w.-]+)/
-        (?<repo>[\w.-]+)
-        (?<path>/[^\@]+)?
-        @(?<ref>.+)
-      }x
-
```

Comment:
Just moved into PackageManager as a constant

Reply:
TBH, this one feels even more generic, like it could be a top-level constant in utils or something. Package manager is fine too, just highlighting this is super generic.

Reply:
I generally try to add constants into package managers that will provide general information. But I can create another module that used for constants in ecosystem such as `constants.rb`